StripeConnect/StripeConnectTests/Internal/Webview/MessageHandlers/OpenSecureWebViewMessageHandlerTests.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  OpenSecureWebViewMessageHandlerTests.swift
+//  StripeConnectTests
+//
+//  Created by Chris Mays on 8/14/24.
+//
+
+@testable import StripeConnect
+import XCTest
+
+class OpenSecureWebViewMessageHandlerTests: ScriptMessageHandlerTestBase {
+    func testMessageSend() {
+        let expectation = self.expectation(description: "Message received")
+        let url = "acct_1234"
```

Comment:
nit: Should this be a realistic URL? I think it might have to be if we change the type to `URL` instead of `String`
```suggestion
        let url = "stripe-connect://some-url"
```